### DISCUSSION TITLE
bootstrap.bundle.min.js

### DISCUSSION THREAD

**Discussion by WTDOsama** [twbs/bootstrap#38555] [Q&A]
when I prepair bootstrap file to work I find problem in (bootstrap.bundle.min.js) if you are put link like before responsive not working you must be add like this (bootstrap. bundle.min.js) with space between dote(.) and bundle . that right or Iam rong???
![1](https://user-images.githubusercontent.com/83521510/236403924-1660b250-508e-491d-9458-9684df3f4ad8.PNG)

**Comment by JamesVanWaza** [comment]
@WTDOsama it should not have any spaces.
